cli/cmd/cluster.go

```diff
@@ -28,11 +28,22 @@ func createCobraCommand(use string, short string, cmd *command) *cobra.Command {
 		Use:   use,
 		Short: short,
 		Run: func(cobraCmd *cobra.Command, args []string) {
-			clusterData, err := doSetup()
-			if err == nil {
-				err = clusterRun(cmd, clusterData)
+			connection, err := connectToGPDB()
+			if err != nil {
+				os.Exit(1)
 			}
```

Comment:
```suggestion
			exitIfError(err)
```
this block repeats 3 times, let's extract it to a function